jujugui/static/gui/src/app/components/user-profile/create-model-button/test-create-model-button.js

```diff
@@ -32,19 +32,19 @@ describe('CreateModelButton', () => {
     controllerAPI = {
       findFacadeVersion: sinon.stub(),
       get: sinon.stub().returns('default'),
-      createModel: (modelName, userName, callback) => {
+      createModel: (modelName, userTag, args, callback) => {
         assert.equal(modelName, 'newmodelname', 'model name not set properly');
-        assert.equal(userName, 'test-owner', 'user name not set properly');
+        assert.equal(userTag, 'user-dalek', 'user tag not set properly');
```

Comment:
Ha ha, OK, I admit, 'user-dalek' is much more fun than 'test-owner'. Good change :wink: